packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx

```diff
@@ -274,100 +275,89 @@ class UserReviewFormComponent extends React.Component<
-      section,
-      userId,
-      formData,
-      goToTeamUserList,
-      userDetails,
-      offlineCountryConfiguration
-    } = this.props
-    let title: string | undefined
-    let actionComponent: JSX.Element
-    const locationId = formData['registrationOffice']
-    const locationDetails =
-      offlineCountryConfiguration['locations'][`${locationId}`] ||
-      offlineCountryConfiguration['facilities'][`${locationId}`] ||
-      offlineCountryConfiguration['offices'][`${locationId}`]
-    if (userId) {
-      title = intl.formatMessage(sysAdminMessages.editUserDetailsTitle)
-      actionComponent = (
-        <SuccessButton
-          id="submit-edit-user-form"
-          disabled={
-            (this.props.formData.systemRole === 'LOCAL_REGISTRAR' ||
-              this.props.formData.systemRole === 'NATIONAL_REGISTRAR') &&
-            !this.props.formData.signature
-          }
-          onClick={this.handleSubmit}
-          icon={() => <Check />}
-          align={ICON_ALIGNMENT.LEFT}
-        >
-          {intl.formatMessage(buttonMessages.confirm)}
-        </SuccessButton>
-      )
-    } else {
-      title = section.title && intl.formatMessage(section.title)
-      actionComponent = (
-        <Button
-          id="submit_user_form"
-          type="positive"
-          size="large"
-          fullWidth
-          disabled={
-            (this.props.formData.systemRole === 'LOCAL_REGISTRAR' ||
-              this.props.formData.systemRole === 'NATIONAL_REGISTRAR') &&
-            !this.props.formData.signature
-          }
-          onClick={this.handleSubmit}
-        >
-          {intl.formatMessage(messages.createUser)}
-        </Button>
-      )
-    }
-    return (
-      <ActionPageLight
-        title={title}
-        goBack={this.props.goBack}
-        goHome={() =>
-          locationDetails
-            ? goToTeamUserList(locationDetails.id)
-            : userDetails?.primaryOffice?.id &&
-              goToTeamUserList(userDetails.primaryOffice.id)
+  let title: string | undefined
+  let actionComponent: JSX.Element
+  const locationId = formData['registrationOffice']
+  const locationDetails =
+    offlineCountryConfiguration['locations'][`${locationId}`] ||
+    offlineCountryConfiguration['facilities'][`${locationId}`] ||
+    offlineCountryConfiguration['offices'][`${locationId}`]
+  if (userId) {
+    title = intl.formatMessage(sysAdminMessages.editUserDetailsTitle)
+    actionComponent = (
+      <SuccessButton
+        id="submit-edit-user-form"
+        disabled={
+          (formData.systemRole === 'LOCAL_REGISTRAR' ||
+            formData.systemRole === 'NATIONAL_REGISTRAR') &&
+          !formData.signature
+        }
+        onClick={handleSubmit}
+        icon={() => <Check />}
+        align={ICON_ALIGNMENT.LEFT}
+      >
+        {intl.formatMessage(buttonMessages.confirm)}
+      </SuccessButton>
+    )
+  } else {
+    title = section.title && intl.formatMessage(section.title)
```

Comment:
Hi @Eezi it seems like `section` is undefined here, you may have have forgoten to destructure it from `props`